<launch>
    <node name="xic_stereo" pkg="stereo_xi_camera" type="stereo_xi_camera_node" output="screen">
        <param name="pXICameraSN_0" type="str" value="CUCAU1814020" />
        <param name="pXICameraSN_1" type="str" value="CUCAU1814018" />
        <param name="pAutoGainExposurePriority" type="double" value="1.0" />
        <param name="pAutoExposureTopLimit" type="int" value="50" />
        <param name="pTotalBandwidth" type="int" value="2400" />
        <param name="pBandwidthMargin" type="int" value="10" />
        <param name="pLoopRate" type="int" value="5" />
        <!-- Warning: Setting the following flag as 1 will enable continous images writing to the file system. -->
        <param name="pFlagWriteImage" type="int" value="0" />
    </node>
</launch>